course_catalog/utils.py

```diff
@@ -176,3 +177,23 @@ def load_course_blacklist():
         response.raise_for_status()
         return [str(line, "utf-8") for line in response.iter_lines()]
     return []
+
+
+def load_course_duplicates(platform):
```

Comment:
This should probably be in `deduplication.py` instead since it doesn't load anything into the database.

Reply:
I think it makes sense for this to live in the same place as `load_course_blacklist` but moving it to `deduplication.py` would be fine too